CHANGELOG.md

```diff
@@ -3,17 +3,22 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Dev]
-### Fixed
-- Starting loading before play (now loads when media element requests data)
 
 ## [Unreleased]
 ### Changed
+- `console.error` replaced by `console.warn` for logging lack of hls.js support.
+
+## [0.1.0] - 2017-04-10
+### Changed
 - Depend on hls.js v0.6.21
 
 ## [0.0.8] - 2016-08-11
 ### Changed
 - Dumping full dependencies tree to package.lock
 
+### Fixed
+- Starting loading before play (now loads when media element requests data)
```

Comment:
For some reason it stayed in `Dev` section, but the feature was merged into master long ago.